admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala

```diff
@@ -158,29 +159,29 @@ object DelegateApiHandler {
     def writeBuf[T](t: T): Buf = Buf.ByteArray.Owned(mapper.writeValueAsBytes(t))
   }
 
+  def getDelegateRsp(dtab: String, path: String, interpreter: NameInterpreter): Future[Response] =
```

Comment:
we should probably move this file from admin to iface if we're going to use it in HttpControlService